website/docs/r/logs_pipeline.html.markdown

```diff
@@ -0,0 +1,254 @@
+            target = "my_arithmetic"
+            is_replace_missing = true
+        }
+    }
+    processor {
+        attribute_remapper {
+            name = "sample attribute processor"
+            is_enabled = true
+            sources = ["db.instance"]
+            source_type = "tag"
+            target = "db"
+            target_type = "tag"
+            preserve_source = true
+            override_on_conflict = false
+        }
+    }
+    processor {
+        category_processor {
+            name = "sample category processor"
+            is_enabled = true
+            target = "foo.severity"
+            category {
+                name = "debug"
+                filter {
+                    query = "@severity: \".\""
+                }
+            }
+            category {
+                name = "verbose"
+                filter {
+                    query = "@severity: \"-\""
+                }
+            }
+        }
+    }
+    processor {
+        date_remapper {
+            name = "sample date remapper"
+            is_enabled = true
+            sources = ["_timestamp", "published_date"]
+        }
+    }
+    processor {
+        message_remapper {
+            name = "sample message remapper"
+            is_enabled = true
+            sources = ["msg"]
+        }
+    }
+    processor {
+        status_remapper {
+            name = "sample status remapper"
+            is_enabled = true
+            sources = ["info", "trace"]
+        }
+    }
+    processor {
+        trace_id_remapper {
+            name = "sample trace id remapper"
+            is_enabled = true
+            sources = ["dd.trace_id"]
+        }
+    }
+    processor {
+        service_remapper {
+            name = "sample service remapper"
+            is_enabled = true
+            sources = ["service"]
+        }
+    }
+    processor {
+        grok_parser {
+            name = "sample grok parser"
+            is_enabled = true
+            source = "message"
+            grok {
+                support_rules = ""
+                match_rules = "Rule %%{word:my_word2} %%{number:my_float2}"
+            }
+        }
+    }
+    processor {
+        pipeline {
+            name = "nested pipeline"
+            is_enabled = true
+            filter {
+                query = "source:foo"
+            }
+            processor {
+                url_parser {
+                    name = "sample url parser"
+                    sources = ["url", "extra"]
+                    target = "http_url"
+                    normalize_ending_slashes = true
+                }
+            }
+        }
+    }
+    processor {
+        user_agent_parser {
+            name = "sample user agent parser"
+            is_enabled = true
+            sources = ["user", "agent"]
+            target = "http_agent"
+            is_encoded = false
+        }
+    }
+}
+```
+## Example Usage: Datadog logs pipeline order
+
+```hcl
```

Comment:
Because it's another resource type, should not it be in another documentation file?